### DISCUSSION TITLE
Add to the benchmark

### DISCUSSION THREAD

**Discussion by cyrusmsk** [saviorand/lightbug_http#12] [Ideas]
TechEmpower quite complex and not very consistent benchmark.
Maybe it will be easier to check the performance if to add here https://github.com/the-benchmarker/web-frameworks

**Comment by saviorand** [comment]
@cyrusmsk thanks for sharing! 
I was able to implement the TechEmpower benchmark. Not publishing currently as I think the support for async handling/parallel connections needs to be improved in Lightbug before that. 
This benchmark looks interesting as well, will look into implementing it.

**Comment by saviorand** [comment]
Looks like it's PHP/Ruby based? I'm not too familiar with these technologies, will appreciate if someone can contribute. Closing this for now
